rasa_nlu/classifiers/sklearn_intent_classifier.py

```diff
@@ -93,8 +93,11 @@ def train(self, training_data, config, **kwargs):
             y = self.transform_labels_str2num(labels)
             X = np.stack([example.get("text_features") for example in training_data.intent_examples])
 
+            config_dict = config.as_dict()
```

Comment:
I think you should be able to do `config.get("sklearn_c", xyz)` directly as the Config class should be a wrapper around a dict (so no need for the `.as_dict()`).